server/src/concepts/height/repositories/commands.js

```diff
@@ -0,0 +1,17 @@
+import Height from "../model/Height.js";
+
+const createHeight = async (data) => {
+  const newHeight = new Height({
+    ...data,
+  });
+
+  try {
+    await newHeight.save();
+    return newHeight;
+  } catch (err) {
+    console.log(err);
+    return "error";
+  }
```

Comment:
I think, it would be better to move try-catch to the next level (where this function is called)

Reply:
It is good to have it here, save can throw some error and. it is fine to catch it here

Reply:
It is fine to have it here, save() can throw some error and it is fine to catch it immediately here